less/mixins/grid-framework.less

```diff
@@ -48,7 +48,7 @@
     width: percentage((@index / @grid-columns));
 
     .row-equal-height & {
-      width: -webkit-calc(percentage((@index / @grid-columns)) ~'- 1px');
+      width: calc(percentage((@index / @grid-columns)) ~'- 1px');
```

Comment:
This is going back some.  But I believe this was done very deliberately as there was a bug that only affected safari.  We don't normally want to reduce the size, but a safari bug caused the final column to wrap, by using webkit-calc we could reduce the size on just that browser and fix the bug.

Reply:
I'd suggest trying safari to see if the bug still exists.  We would typically use this class when we have target areas like 'tiles' as they should be equal height e.g. https://bootstrap.transferwise.com/components/#tile-rows

![image](https://user-images.githubusercontent.com/1318111/86927431-3f6bb800-c12b-11ea-9293-225cff7c9a15.png)

Reply:
hahah that's good to know, I'll double check

Reply:
I think the bug still exists 
![image](https://user-images.githubusercontent.com/1318111/86928496-873f0f00-c12c-11ea-9e5b-25851d581393.png)

![image](https://user-images.githubusercontent.com/1318111/86928528-91610d80-c12c-11ea-8733-9cf580e1ed6d.png)

Reply:
Yep looking at that as well.
Worth noting though that the -webkit- prefix also applies in Chrome. So at this point I'd still consider removing the prefix as a large part of users already have this behavior

Reply:
While this wouldn't be ideal, it's barely noticable. Also the tile is being used in a very small number of places like the profile and recipient selection screen which are being redesigned right now. I'd say the downsides are low

Reply:
We could also change the 1x to 0.5px in order to make it even less noticeable. Comparison in safari:

<img width="317" alt="Screenshot 2020-07-08 at 17 21 10" src="https://user-images.githubusercontent.com/43746750/86930283-8499e500-c13f-11ea-87f6-537ff033f4af.png">

<img width="296" alt="Screenshot 2020-07-08 at 17 20 53" src="https://user-images.githubusercontent.com/43746750/86930277-84014e80-c13f-11ea-8228-b82ef542322f.png">

WDYT?

Reply:
Yeah, I'm fine with that as long as you check in browser.  The best solution here would be to switch the Bootstrap grid to use flex instead of float.  I think safari is confused by the floated columns being used at the same time as the flexbox height.  But I have no idea how complex that would be.